resources/migrations/001_update_migrations.yaml

```diff
@@ -5152,6 +5152,38 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
```

Comment:
Will need to check with BE if it makes more sense to just combine these migrations or if they need downgrades or different migrations for different DBs